packages/javascript-sdk/project.json

```diff
@@ -19,13 +19,12 @@
       "options": {
         "compiler": "babel",
         "entryFile": "packages/javascript-sdk/src/index.ts",
-        "outputPath": "samples/_static/js",
+        "outputPath": "packages/javascript-sdk/bundles",
         "deleteOutputPath": true,
         "project": "packages/javascript-sdk/package.json",
         "tsConfig": "packages/javascript-sdk/tsconfig.lib.json",
         "cwd": "packages/javascript-sdk",
-        "parallel": true,
-        "rollupConfig": "packages/javascript-sdk/rollup.config.js"
```

Comment:
This is no longer needed because each app has its own build, and no longer relies on the one built artifact produced by this rollup, right?

Reply:
Correct, the embedded login and centralized login will bundle the sdk on it's own, so we no longer care about rollup outputting to the _static directory, it's not required.

We can just use rollup to bundle the sdk for delivery and not worry about outputting it around the repo.